run_simulator.sh

```diff
@@ -6,6 +6,10 @@ export FLUX_SCHED_RC_NOOP=1
 # already. These assume that flux-core, flux-sched, and this aux repo
 # are all sibling directories
 #
+# module load python/2.7
+# module load python-cffi/1.1.2
+# module load python-pycparser/2.10
+export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$HOME/local2/lib/pkgconfig"
```

Comment:
@sampollard, thanks for creating this PR.  Changes generally look great! Do you mind removing this specific export line?